### PR TITLE
Prepare for 7.0.0

### PR DESCRIPTION
**Pull request by Dridi**
:tada:

### CODE REVIEW

**Comment by Dridi** [comment]
Added a missing step to the release process: https://github.com/varnishcache/varnish-cache/wiki/Release-procedure/_compare/aca2acd391e31b0c4e1605716203005a9ba8c22c...1828e403030ed12ccfe4a00c8d65fa28f4fc5727

File: R1/source/index.rst [hunk 1]
```diff
@@ -58,7 +67,7 @@ Please see :ref:`VSV00006`.
 2021-03-15 - Varnish 6.6.0 is released
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Our bi-annual"fresh" release is here:  :ref:`rel6.6.0`
+Our bi-annual "fresh" release is here:  :ref:`rel6.6.0`
 
 (The 2021-09-15 release is likely to be 7.0.0)
```
Comment (hermunn) [inline R1/source/index.rst:72]: This line is now ripe for harvest. We know that today's release is 7.0. Or maybe not, since this is a news item...

**Review by hermunn — APPROVED** [review]
Apart from my single comment (which seems to be wrong), I have no complaints. Great work!

File: R1/source/index.rst (hunk 1 above)
Comment (Dridi) [reply R1/source/index.rst:72]: I'm only adding a missing space to an old entry I happened to copy-paste for the new entry :)

Comment (hermunn) [reply R1/source/index.rst:72]: I was commenting on the line 63/72, which is not part of the diff, but just under your changed line. No matter, let's leave it as it is, and celebrate that we have a release out!

**Pull request merged**